### PR TITLE
feat(separator): Adding new component - FRONT-4597

### PR DESCRIPTION
**Pull request by planctus**
Well, in the end i talked to EWPP to understand what to do and they are expecting a template, so i have added both a vanilla and a twig public packages.
The demo in storybook is showing the "separator" in a kind of real example with some content, in the website we show only the hr, instead.
An example has been added also to the html default tags story

### CODE REVIEW

File: src/implementations/twig/components/separator/separator.story.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  },
+};
+
+export const Default = (_, { loaded: { component } }) => component;
+
+Default.render = async () => {
+  const renderedSeparator = `<ul class="ecl-unordered-list">
+      <li class="ecl-unordered-list__item">Unordered list</li>
+      <li class="ecl-unordered-list__item">Unordered list 
+        <ul>
```
Comment (emeryro) [inline src/implementations/twig/components/separator/separator.story.js]:
```suggestion
        <ul class="ecl-unordered-list">
```

File: src/implementations/twig/components/separator/separator.story.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+          <li class="ecl-unordered-list__item">Nested unordered list</li>
+          <li class="ecl-unordered-list__item">Nested unordered list</li>
+        </ul>
+      </li>
+    </ul>
+    ${await separator()}
+    <ol class="ecl-ordered-list">
+      <li class="ecl-ordered-list__item">Ordered list</li>
+      <li class="ecl-ordered-list__item">Ordered list <ol>
+          <li class="ecl-ordered-list__item">Nested ordered list</li>
```
Comment (emeryro) [inline src/implementations/twig/components/separator/separator.story.js:33]:
```suggestion
        <ol class="ecl-ordered-list">
          <li class="ecl-ordered-list__item">Nested ordered list</li>
```

File: src/implementations/vanilla/components/separator/README.md
Comment (emeryro) [inline src/implementations/vanilla/components/separator/README.md:1]: We should have at least the component name here

**Review by emeryro — CHANGES_REQUESTED** [review]
Just a few small things

Also, not related to this PR, but do you know why we have some padding below the unordered list (and not the other lists)?
image

**Comment by planctus** [comment]
no, i don't, but i have created a ticket about this: https://citnet.tech.ec.europa.eu/CITnet/jira/browse/FRONT-4602

**Review by emeryro — APPROVED** [review]

**Pull request merged**